tests/test_client_response.py

```diff
@@ -424,6 +424,30 @@ def side_effect(*args, **kwargs):
     assert response.get_encoding().lower() in ('windows-1251', 'maccyrillic')
 
 
+async def test_get_encoding_body_none(loop, session) -> None:
+    response = ClientResponse('get', URL('http://def-cl-resp.org'),
+                              request_info=mock.Mock(),
+                              writer=mock.Mock(),
+                              continue100=None,
+                              timer=TimerNoop(),
+                              traces=[],
+                              loop=loop,
+                              session=session)
+
+    def side_effect(*args, **kwargs):
+        fut = loop.create_future()
+        fut.set_result('{"encoding": "test"}')
+        return fut
+
+    response._headers = {'Content-Type': 'text/html'}
+    content = response.content = mock.Mock()
+    content.read.side_effect = side_effect
+
+    with pytest.raises(RuntimeError):
```

Comment:
```suggestion
    with pytest.raises(
        RuntimeError,
        match='^Cannot guess the encoding of a not yet read body$',
    ):
```

Reply:
Updated PR, thank's for review
Does it always make sense to check exception message?